static/schema/projects.json

```
@@ -76,6 +81,12 @@
                     "items": {
                         "$ref": "#/$defs/discussionLink"
                     }
+                },
+                "issueNumbers": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/$defs/issueNumber"
+                    }
```

Perhaps this should be `issueLinks` so that it's not sepcific to Jira?

```suggestion
                "issueLinks": {
                    "type": "array",
                    "items": {
                        "$ref": "#/$defs/issueLinks"
                    }
```